### DISCUSSION TITLE
T-Display-S3 board define for Arduino

### DISCUSSION THREAD

**Discussion by git2212** [espressif/arduino-esp32#8164] [General]
Is there a board define specific to the T-Display-series for the Arduino platform?... need to be able to dinstinguish between ESP32S3 generic and this family of products 

Thx

**Comment by mrengineer7777** [comment]
Yes. https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L2989

**Comment by git2212** [comment]
I meant something I can check inside a sketch with an #ifdef ?... CONFIG_IDF_TARGET_ESP32S3 is present but it's not specific to the T-Display-series...
